internal/v2/s2av2_test.go

```diff
@@ -1,12 +1,19 @@
 package v2
 
 import (
+	"flag"
 	"testing"
 	"context"
 	"github.com/google/go-cmp/cmp"
 	"google.golang.org/protobuf/testing/protocmp"
 )
 
+var (
+	fakes2av2Address = flag.String("address", "0.0.0.0:8008", "Fake S2Av2 address")
```

Comment:
The fakes2av2_server is set to run at :8008; in these unit tests, I don't setup a fake service because the tests never reach the point of actually dialing and creating a stream to fakes2av2, since the unit tests are only testing the failure case. Now that I think about it, I could just use :0 here, not hardcode the actually port the fakes2av2 server runs on, because if we ever do change the unit tests to test the success case of the handshake(actually dial and setup stream), then we will have toe startup and tear down a fake server in the test. 

Also, thanks for pointing it out, no need for it to be a flag. 

To summarize:
- I'll change the port to :0 
- I'lll make the port a global var, not a flag.

Thanks!